docs/root/start/sandboxes/cache.rst

```diff
@@ -0,0 +1,289 @@
+For the purposes of the demo, a response's date of creation is appended to its body before being served. 
+An Etag is computed for every response for validation purposes, which only depends on the response body in the yaml file (i.e. the appended date is not taken into account). 
+Cached responses can be identified by having an ``age`` header. Validated responses can be identified by having a generation date older than the ``date`` header;
+as when a response is validated the ``date`` header is updated, while the body stays the same.
+
+Running the Sandbox
+~~~~~~~~~~~~~~~~~~~
+
+The following documentation runs through the setup of an Envoy cluster organized
+as is described in the image above.
+
+**Step 1: Install Docker**
+
+Ensure that you have a recent versions of ``docker`` and ``docker-compose`` installed.
+
+A simple way to achieve this is via the `Docker Desktop <https://www.docker.com/products/docker-desktop>`_.
+
+**Step 2: Clone the Envoy repo**
+
+If you have not cloned the Envoy repo, clone it with:
+
+``git clone git@github.com:envoyproxy/envoy``
+
+or
+
+``git clone https://github.com/envoyproxy/envoy.git``
+
+**Step 3: Start all of our containers**
+
+.. code-block:: console
+
+    $ pwd
+    envoy/examples/cache
+    $ docker-compose build --pull
+    $ docker-compose up -d
+    $ docker-compose ps
+
+           Name                      Command            State                             Ports                          
+    ------------------------------------------------------------------------------------------------------------------------
+    cache_front-envoy_1   /docker-entrypoint.sh /bin ... Up      10000/tcp, 0.0.0.0:8000->8000/tcp, 0.0.0.0:8001->8001/tcp
+    cache_service1_1      /bin/sh -c /usr/local/bin/ ... Up      10000/tcp, 8000/tcp                                      
+    cache_service2_1      /bin/sh -c /usr/local/bin/ ... Up      10000/tcp, 8000/tcp    
+
+**Step 4: Test Envoy's HTTP caching capabilities**
+
+You can now send a request to both services via the ``front-envoy``. Note that since the two services have different routes,
+identical requests to different services have different cache entries (i.e. a request sent to service 2 will not be served by a cached
+response produced by service 1).
+
+To send a request: 
+
+``curl -v localhost:8000/service/<service_no>/<response>``
+
+``service_no``: The service to send the request to, 1 or 2.
+
+``response``: The response that is being requested. The responses are found in :repo:`/examples/cache/responses.yaml`.
+
+You can change the responses headers and bodies (or add new ones) while the sandbox is running to experiment.
+
+Example responses
+^^^^^^^^^^^^^^^^^^^^^
+1. A response that stays fresh for a minute:
+
+.. code-block:: console
+
+    $ curl -v localhost:8000/service/1/valid-for-minute
```

Comment:
i might be misunderstanding, but it seems to cache for longer than a minute

Reply:
How so?

Reply:
The entry is kept on cache for more than a minute, but it only stays fresh for a minute. After a minute passes, the Cache Filter performs validation before serving the response. If it is found to be still fresh (body not changed), then it serves the cached response. Otherwise, it serves the new response.

Reply:
I will update the docs to list the example responses with a short description.

Reply:
ok, thanks for the explanation - i wasnt clear straight away